scripts/experimental/test_callable_avl.py

```diff
@@ -0,0 +1,78 @@
+# Tim Momose, January 2015
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+import pylab as plt
+from copy import deepcopy
+# SUAVE Imports
+from SUAVE.Attributes import Units
+from full_setup_737800 import full_setup_737800
+# SUAVE-AVL Imports
+from SUAVE.Methods.Aerodynamics.AVL.AVL_Data.AVL_Cases    import AVL_Run_Case, AVL_Cases
+from SUAVE.Methods.Aerodynamics.AVL.AVL_Callable          import AVL_Callable
+
+
+# -------------------------------------------------------------
+#  Setup function
+# -------------------------------------------------------------
+
+def setup_avl_test(vehicle):
+
+    default_case = AVL_Run_Case()
+    default_case.conditions.freestream.mach     = 0.2
+    default_case.conditions.freestream.velocity = 150 * Units.knots
+    default_case.conditions.aerodynamics.parasite_drag = 0.0177
+
+    avl_instance = AVL_Callable()
+    avl_instance.initialize(vehicle)
+
+    for wing in vehicle.wings:
+        for cs in wing.control_surfaces:
+            default_case.append_control_deflection(cs.tag,0.0) # default all control surfaces to zero deflection
+
+    return avl_instance, default_case
+
+
+# -------------------------------------------------------------
+#  Test Script
+# -------------------------------------------------------------
+
+# Set up test defaults
```

Comment:
can you please write this test script in a ```main()``` function and put it at the top?  check out some of the other scripts for an example